qiskit/ignis/mitigation/measurement/circuits.py

```diff
@@ -16,43 +16,52 @@
 Measurement calibration circuits. To apply the measurement mitigation
 use the fitters to produce a filter.
 """
-
+from typing import List, Tuple
 from qiskit import QuantumRegister, ClassicalRegister, \
     QuantumCircuit, QiskitError
 from ...verification.tomography import count_keys
 
 
-def complete_meas_cal(qubit_list=None, qr=None, cr=None, circlabel=''):
+def complete_meas_cal(qubit_list: List[int] = None,
+                      qr: List[QuantumRegister] = None,
+                      cr: List[ClassicalRegister] = None,
+                      circlabel: str = ''
+                      ) -> Tuple[List[QuantumCircuit], List[str]
+                                 ]:
     """
     Return a list of measurement calibration circuits for the full
     Hilbert space.
 
     Each of the 2**n circuits creates a basis state
```

Comment:
2**n should be math.
Also say that n is the number of qubits. Something like: 

> If there are n qubits then there are 2**n calibration circuits that each one of them creates a basis state.

Reply:
done